### DISCUSSION TITLE
Do we need to change the minimum level for Microsoft.AspNetCore to Warning in your logger configuration or appsettings.json file?

### DISCUSSION THREAD

**Discussion by VictorioBerra** [Dotnet-Boxed/Templates#524] [Q&A]
### Question

Serilog docs: https://github.com/serilog/serilog-aspnetcore#request-logging

> To enable the middleware, first change the minimum level for Microsoft.AspNetCore to Warning in your logger configuration or appsettings.json file:

```
.MinimumLevel.Override("Microsoft.AspNetCore", LogEventLevel.Warning)
```

Is there a reason we have not done this? Is this not required?

### Project template

All.

## Environment

All.

**Comment by RehanSaeed** [comment]
Good point. Want to create a PR to fix this?

**Comment by VictorioBerra** [comment]
Yeah ill do it.

**Comment by VictorioBerra** [comment]
Actually, you already have Microsoft overriden to warning. I think that will satisfy the requirement because it will change anything in `Microsoft.*`.
